### PR TITLE
[ism8] Add channel types for heat pumps

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Extend add-on to support new information provided by lates ISM firmware versions 1.80 and 1.90.

* Add channel type power-r used by CHA device
* Add channel type active-energy-r used by solar module
* Add channel type number-r used for device information
* Extend channel type volumetricflow-r to data type used by solar module
* i18n
* Rework documentation

Refs:  #17682

### CODE REVIEW

**Comment by holgerfriedrich** [comment]
Tests to be extended.

Live test on ISM8 not yet done. I can do some testing of the new channels next week.

File: bundles/org.openhab.binding.ism8/README.md [hunk 1]
```diff
@@ -49,28 +53,34 @@ Description:
...
+Not all available data types of the ISM8 interface are fully supported.
+For the moment, the following data types are implemented:
+
+| Channel type   | Datapoint type                           | Item type                 | R/W | KNX-type's                 |
+|----------------|------------------------------------------|---------------------------|-----|----------------------------|
+| switch-rw      | Digital DataPoint                        | Switch                    | R/W | 1.001, 1.002, 1.003, 1.009 |
+| switch-r       | Digital Readonly DataPoint               | Switch                    | R   | 1.001, 1.002, 1.003, 1.009 |
+| percentage-rw  | Percentage  DataPoint                    | Number:Dimensionless      | R/W | 5.001                      |
+| percentage-r   | Percentage Readonly DataPoint            | Number:Dimensionless      | R   | 5.001                      |
+| value1ucount-r | Value 2-byte Unsigned Readonly DataPoint | Number:Dimensionless      | R   | 5.010                      |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.ism8/README.md:73]: Can `value1ucount-r` and `value2ucount-2` channel types be combined to a single `nummeric-r` channel type? As both are numbers, just the size is different? Also a little less technicall notation.

There is also a typo:
```suggestion
| value1ucount-r | Value 1-byte Unsigned Readonly DataPoint | Number:Dimensionless      | R   | 5.010                      |
```

File: bundles/org.openhab.binding.ism8/README.md [hunk 2]
```diff
@@ -49,28 +53,34 @@ Description:
...
+| value2ucount-r | Value 2-byte Unsigned Readonly DataPoint | Number:Dimensionless      | R   | 7.001                      |
+| temperature-rw | Temperature DataPoint                    | Number:Temperature        | R/W | 9.001,9.002                |
+| temperature-r  | Temperature Readonly DataPoint           | Number:Temperature        | R   | 9.002,9.002                |
+| pressure-r     | Pressure Readonly DataPoint              | Number:Pressure           | R   | 9.006                      |
+| flowrate-r     | Flowrate Readonly DataPoint              | Number:VolumetricFlowRate | R   | 9.025, 13.002              |
+| activeenergy-r | Active Energy Readonly DataPoint         | Number:Energy             | R   | 13.010, 13.013             |
```
Comment (lsiepel) [inline bundles/org.openhab.binding.ism8/README.md:79]: words should be seperated with a dash: `activeenergy-r` => `active-energy-r`

```suggestion
| active-energy-r | Active Energy Readonly DataPoint         | Number:Energy             | R   | 13.010, 13.013             |
```

**Review by lsiepel — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.ism8/README.md (hunk 1 above)
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.ism8/README.md]: yes, it is now combined in one number-r channel.
The add-on anyway distinguishes the processing steps depending on the given DPT.

File: bundles/org.openhab.binding.ism8/README.md (hunk 2 above)
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.ism8/README.md]: done

**Comment by holgerfriedrich** [comment]
Tests have now been added and cover reading all implemented read data points.

Live test on ISM8 not yet done. I can do some testing of the new channels next week.

**Review by lsiepel — APPROVED** [review]
Thanks LGTM. Will wait for live test to be confirmed.

**Comment by holgerfriedrich** [comment]
@lsiepel Thank you. I think we can proceed, I tested successfully on live system.
CHA data (power channels) work fine. Device info works as well.

I could not test the new channels for solar module, as the system under test did not provide related datapoints.

**Comment by lsiepel** [comment]
> @lsiepel Thank you. I think we can proceed, I tested successfully on live system. CHA data (power channels) work fine. Device info works as well.
> 
> I could not test the new channels for solar module, as the system under test did not provide related datapoints.

Risk is minimal, let’s go

**Pull request merged**